server/controllers/roomsController.js

```diff
@@ -1,184 +1,175 @@
 const roomsController = {};
 
-// const Room = require("../models/Room");
-
-const rooms = [
+let rooms = [
```

Comment:
ESLint shouts: 'This is WRONG' because it wants a const. However, rooms will change when applying the PATCH :/